com.unity.netcode.gameobjects/Runtime/SceneManagement/NetworkSceneManager.cs

```diff
@@ -224,16 +214,46 @@ public class NetworkSceneManager : IDisposable
         /// </summary>
         public void Dispose()
         {
-            SceneEventData.Dispose();
-            SceneEventData = null;
-            ClientSynchEventData.Dispose();
-            ClientSynchEventData = null;
+            foreach (var keypair in SceneEventDataQueue)
+            {
```

Comment:
a (warning?) log here would be useful in case of leaving some events unprocessed. it could help us when we debug things.

Reply:
Yeah, I thought about that but there is a chance that someone just shutdowns the application in the middle of a scene event.
So, I guess I could at a minimum log SceneEventData object entries being disposed.

Reply:
it could be lower than a warning log. we had something like `NetworkLogLevel.Develop` or `NetworkLogLevel.Debug`, that'd become handy here. minor enough, I'll leave it up to you.

Reply:
Actually, I like it at the normal level because in all reality the only times that should log anything is if you are exiting in the middle of a scene event.  Any other scenario it should not log anything.